qubes-rpc/qubes-sync-clock

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/python3
+# -*- encoding: utf8 -*-
+#
+# The Qubes OS Project, http://www.qubes-os.org
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+
+import sys
+import re
+import subprocess
+
+def main():
+    stdin = sys.stdin.read(25)
+
+    date_out = stdin.strip()
```

Comment:
I think this should be called untrusted_date_out to highlight that the data is untrusted. Shouldn't it? @marmarek